tst/regression/test_suites/restart/parthinput.restart

```diff
@@ -39,9 +39,6 @@ x3max = 1
 ix3_bc = periodic
 ox3_bc = periodic
 
-derefine_count = 999999 # disable this since derefinement counter resets on
-                        # restart and thus leads to different results
-
```

Comment:
I am not sure how this got into the restart test.

Reply:
I think it was there from the beginning but I thought you removed before. :shrug:

Reply:
So what does this mean if the tests still pass independent of this variable?
Is our tolerance too loose?
I somehow assumed that this, in principle, you result in a failed test.
I'm going to trigger the full suite to see what happens.

Reply:
By setting the `derefine_count` to a huge number, it was ensured that derefinement was never triggered. This meant that restarts didn't depend on on the derefinement counter of each block and were exact in the test. Now that we store the derefinement counter in the restart files, `derefine_count` can be set to any number and the restarts will still be exact (while previously it would not be unless it was set to zero or a huge number).

Reply:
Ah, I missed that the restart test does not compare to our (precalculated) gold files but to a freshly generated state, so, yes, this makes a lot of sense now.